frontend/src/components/misc/KowlTable.tsx

```diff
@@ -560,15 +560,14 @@ export class SearchTitle extends Component<{
         setTimeout(this.focusInput);
 
         return <span>
-            <span >{this.props.title}</span>
-            <div className="tableInlineSearchBox"
+            {!props.observableFilterOpen.filterOpen && <span>{this.props.title}</span>}
+            <Box
```

Comment:
Small suggestion, can just use <Flex> instead of <Box display="flex">, also if it's a Chakra component you don't need `style` object, can just use the props instead.